server/core_benchmarks_test.go

```diff
@@ -249,3 +249,134 @@ func BenchmarkCoreTLSFanOut(b *testing.B) {
 		)
 	}
 }
+
+func BenchmarkCoreTLSFanIn(b *testing.B) {
+	const (
+		subject            = "test-subject"
```

Comment:
Should probably have per-publisher subjects and a wildcard subscription for the consumer